proposals/description/tip_118.ts

```diff
@@ -0,0 +1,175 @@
+        addresses.rai, // token
+        addresses.raiHoldingDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Migrate RAI from the RAI PSM to the RAI holding PCV deposit'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.raiPriceBoundPSM, // pcvDeposit
+        addresses.fei, // token
+        addresses.daiFixedPricePSM, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Migrate FEI from the RAI PSM to the DAI PSM'
+    },
+
+    // 2. Revoke PSM permissions
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('MINTER_ROLE'), addresses.ethPSM],
+      description: 'Revoke MINTER_ROLE from ETH PSM'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('MINTER_ROLE'), addresses.lusdPSM],
+      description: 'Revoke MINTER_ROLE from LUSD PSM'
+    },
+
+    // RAI PSM does not have MINTER_ROLE
+
+    // 3. Pause all unpaused functionality on the PSMs
+    {
+      target: 'raiPriceBoundPSM',
+      values: '0',
+      method: 'pauseRedeem()',
+      arguments: (addresses) => [],
+      description: 'Pause redemptions on the Rai Price bound PSM'
+    },
+
+    // 4. Unset deprecated PSMs as safe addresses
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'unsetSafeAddresses(address[])',
+      arguments: (addresses) => [[addresses.ethPSM, addresses.lusdPSM, addresses.raiPriceBoundPSM]],
+      description: 'Unset the ETH, LUSD and RAI PSMs as safe addresses'
+    },
+
+    // Add new Holding PCV deposits to Collaterization Oracle
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposits(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.wethHoldingDeposit,
+          addresses.lusdHoldingDeposit,
+          addresses.raiHoldingDeposit,
+          addresses.daiHoldingDeposit,
+          addresses.voltHoldingDeposit
+        ]
+      ],
+      description: 'Add the holding ERC20 deposits to the Collaterization Oracle'
+    },
+
+    ///  PCV DRIP CONTROLLERS
+    {
+      target: 'aaveEthPCVDripController',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: 'Pause the ETH PCV drip controller'
+    },
+    {
+      target: 'raiPCVDripController',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: 'Pause the RAI PCV drip controller'
+    },
+    {
+      target: 'lusdPCVDripController',
+      values: '0',
+      method: 'pause()',
+      arguments: (addresses) => [],
+      description: 'Pause the LUSD PCV drip controller'
+    }
```

Comment:
these should have their roles revoked. Same with their skimmers

Reply:
Update: Not shipping Fei Sink and instead using DAI skimmer to burn Fei. Will activate it and burn the Fei. Then deprecate all other skimmers